### ISSUE TITLE
The NVDA is not announcing the control type for the "Microsoft" and "Ratings" controls.

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/vscode#132645]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yMAS;#A11yTCS;#VisualStudioCode-Win32-Apr2021;#Visual Studio Code Client;#A11ySev2;#WCAG4.1.2;#Benchmark;#DesktopApp;#Win10;#NVDA;#JAWS;

### Environment Details:
Visual Studio Code
Azure Function
Operating System: Windows 10 Pro 21H1

### Steps to Reproduce:

1. Launch the Visual studio code.
2. TAB to on "Extension" button and search for "Azure Functions" and select it using ENTER key.
3. "Azure Function" screen should open.
4. Start NVDA and TAB to "Microsoft" control.
5. Observe the announcement for "Microsoft" control.

### Actual:
The NVDA is only announcing "Clickable Microsoft" but it does not announce the control type for this "Microsoft" control.

**Same issue is also observed with "Ratings" control.
This issue is also observed with JAWS.**

### Expected:
The NVDA should announce the appropriate control type for "Microsoft" and "Ratings" controls.

### User Impact:
Screen reader users won't be able to know control type of "Microsoft" and "Ratings" controls.

**Comment by isidorn** [comment]
@12-shweta what control type are you suggesting that "Microsoft" and "Ratings" should be?

**Comment by 12-shweta** [comment]
For "Microsoft", control type should be button and for "Rating", it should be link as after clicking we got redirected to web browser.

**Comment by isidorn** [comment]
@12-shweta this makes good sense, thank you for your suggestion. I have pushed a fix.

**Comment by 12-shweta** [comment]
Verified the bug in the VS code Insiders.
Issue is fixed and now the NVDA announces control type for the controls.
But for the "Microsoft" control it also announces "Microsoft button publisher name"
![Issue](https://user-images.githubusercontent.com/54656883/135384204-b9244f9a-00c0-496d-bf22-06628cbf1244.gif)

**Comment by 12-shweta** [comment]
@isidorn 
Logged an external bug for the unnecessary announcement of "Publisher Name" while navigating the "Microsoft" button
microsoft/vscode#134262

**Comment by isidorn** [comment]
Ok, thanks! I will comment on that one.

**Comment by Rama8345** [comment]
**Github Tags:**#Closed;
